content/docs/reference/config/policy-and-telemetry/metrics.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Default Metrics
+description: Default Metrics exported from Istio through Mixer.
+weight: 50
+---
+
+This page presents details about the metrics that Istio collects when using its initial configuration. You can add and remove metrics by changing configuration at any time, but this is the built-in set. They can be found
+[here](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml)
+under the section with “kind: metric”. It uses [metric
+template](/docs/reference/config/policy-and-telemetry/templates/metric/) to define these metrics.
+
+We will describe metrics first and then the labels for each metric.
+
+## Metrics
+
+*   **Request Count**: This is a `COUNTER`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L786:9)
+    incremented for every request handled by an Istio proxy.
+
+*   **Request Duration**: This is a `DISTRIBUTION`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L802:9)
+    which measures the duration of the request (as observed by the server-side
+    proxy).
+
+*   **Request Size**: This is a `DISTRIBUTION`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L818:9)
+    which measures the size of the HTTP request’s body size.
+
+*   **Response Size**: This is a `DISTRIBUTION`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L834:9)
+    which measures the size of the HTTP response body size.
+
+*   **Tcp Byte Sent**: This is a `COUNTER`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L850:9)
+    which measures the size of total bytes sent during response in case of a TCP
+    connection as measured by the server-side proxy.
+
+*   **Tcp Byte Received**: This is a `COUNTER`
+    [metric](https://github.com/istio/istio/blob/{{<branch_name>}}/install/kubernetes/templates/istio-mixer.yaml.tmpl#L867:9)
+    which measures the size of total bytes received during request in case of a
+    TCP connection as measured by the server-side proxy.
+
+## Labels
+
+*   **Source Service**: This identifies the source service responsible for an
+    incoming request. This is also the FQDN for a source service. Ex:
+    "reviews.default.svc.cluster.local".
+
+    ```yaml
+       #Default Attribute Expression
```

Comment:
For this and the following 3 code blocks:

- Remove the Default Attribute Expression comment line, it's not helpful
- Remove the 3 character indent.

Reply:
Thanks..FIxed